Introduction-to-Grails-Development-with-Intellij/index.md

```diff
@@ -0,0 +1,293 @@
+- Knowledge in Java development
+- Java development kit(JDK) installed in your computer/laptop to help in application development
+
+### Installing Grails
+
+- Download [Grail Framework](https://grails.org/download.html) as per the tutorial we were at version 4.0.6 in which you will get a zipped file.
+- Extract the zipped file and place it where you keep development tools. This is how to do Windows [Unzip](https://www.windowscentral.com/how-zip-and-unzip-files-windows-10). Ubuntu [unzip](https://askubuntu.com/questions/499807/how-to-unzip-tgz-file-using-the-terminal)
+- Set environment variable `GRAILS_HOME` to point where Grails installation directory is located. [Set windows environment variable](https://www.computerhope.com/issues/ch000549.htm).[ Set Ubuntu environment variable](https://askubuntu.com/questions/58814/how-do-i-add-environment-variables)
+
+### Getting Started
+
+Grails provides a set of commands to support application development.IDE has also easen the use of commands and provided a way users can click menus for fast application development.
+
+To create an application in Grail framework `create-app` command can be passed in terminal or windows CMD with arguments like the name of the application.
+
+The command below shows how to create an application via terminal or CMD. Open terminal or CMD in a directory where you want to save the application. In this case, we have a folder called `Projects` in the document folder
+
+Navigate to `Projects` folder via terminal
+
+```bash
+#Ubuntu
+cd C:\Users\user\Documents\Projects
+
+#Windows
+cd C:\Users\user\Documents\Projects
+```
+
+Create the first application with the following command
+
+```bash
+ grails create-app Firstapp
+```
+
+![CREATE-APP](/engineering-education/how-to-replicate-postgresql-database/grails-create-app.png)
+
+The Grails application is formed within the  `Firstapp` directory and many other grails commands can be executed.
+
+To run the application run the following command
+
+```bash
+grails run-app
+```
+The command will make the web application to run via a browser. You can type the highlighted command in the browser.
+
+![CREATE-APP](/engineering-education/how-to-replicate-postgresql-database/grails-run-app.png).
+
+![BROWSER VIEW](/engineering-education/how-to-replicate-postgresql-database/grails-first-web-app.png).
+
+### Folder structure and directories
+
+The grails application has the following folder structure.
+
+```bash
+├── .gradle
+├── build
+├── gradle
+├── grails-app
+├── src
+├── .gitignore.txt
+├── build.gradle
+├── gradle
+├── gradlew
+├── gradlew.bat
+├── grailsw
+├── grailsw.bat
+└── grails-wrapper
+```
+
+In `gails-app`  has  the following directories
+
+```bash
+├── assets
+├── conf
+├── Controllers
+├── domain
+├── i18n
+├── init
+├── services
+├── taglib
+├── utils
+└── views
+```
+
+### assets folder
+
+This is where all static folders are kept like javascript, Images, CSS and other static multimedia content
```

Comment:
Consider rewriting "This is where all static folders are ..." to "This is where all static files are ..."

Reply:
sorted